src/web/screens/keyPairsScreen.tsx

```diff
@@ -0,0 +1,37 @@
+import { Suspense } from 'react';
+import { Await, defer, useLoaderData } from 'react-router-dom';
+
+import { KeyPairModel } from '../components/KeyPairs/KeyPairModel';
+import KeyPairsTable from '../components/KeyPairs/KeyPairsTable';
+import { GetKeyPairs } from '../services/keyPairsServices';
+import { PortalRoute } from './routeUtils';
+
+function Loading() {
```

Comment:
nit: is there a good reason to use this rather than the existing `<Loading>` component in Core?

Reply:
Not a good reason - I just didn't know about it.  Thanks.